.github/workflows/issue-for-config.yml

```diff
@@ -0,0 +1,58 @@
+name: Check PR for configuration changes
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+    - 'features/featureflag_string.go'
+    - 'test/config-next/*.json'
+    - 'test/config-next/*.yaml'
+    - 'test/config-next/*.yml'
+
+jobs:
+  check-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
```

Comment:
It doesn't seem like this step should be necessary? None of the subsequent steps care about file contents.

Reply:
Oh whoops, yeah I left that in from a prior iteration that was running a python script.